assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -620,7 +620,7 @@ che.server.secure_exposer=default
 che.server.secure_exposer.jwtproxy.token.issuer=wsmaster
 che.server.secure_exposer.jwtproxy.token.ttl=8800h
 che.server.secure_exposer.jwtproxy.auth.loader.path=/_app/loader.html
-che.server.secure_exposer.jwtproxy.image=eclipse/che-jwtproxy:latest
```

Comment:
I worry that this is another default property that we will have to maintain. Are we going to have to issue a PR in Che for every PR in jwtproxy?

Reply:
Well, IMHO that's better than breaking every che installation in case of the need of incompat changes in jwtproxy.

Reply:
See the current catastrophe of failing E2E tests because we use theia:latest ;)

Reply:
I agree, but I think down the line we'll really need an automated way of setting these tags on releases.

It's not an issue for this PR, regardless.